iOS_SDK/OneSignalSDK/Source/OSInAppMessageAction.m

```diff
@@ -72,6 +75,8 @@ + (instancetype _Nullable)instanceWithJson:(NSDictionary *)json {
         action.closesMessage = [json[@"close"] boolValue];
     else
         action.closesMessage = true; // Default behavior
+    
+    [OneSignal onesignal_Log:ONE_S_LL_VERBOSE message:[NSString stringWithFormat:@"%@",json]];
```

Comment:
nit indentation comment, space before the json variable.
Is it clear that this log is from the OSInAppMessageAction" If not maybe add @"OSInAppMessageAction json: %@" introduction string

Reply:
fixed